target/linux/ipq806x/image/generic.mk

```diff
@@ -160,6 +160,24 @@ define Device/edgecore_ecw5410
 endef
 TARGET_DEVICES += edgecore_ecw5410
 
+define Device/extreme_ap3935
+	$(call Device/LegacyImage)
+	$(call Device/UbiFit)
+	DEVICE_VENDOR := Extreme Networks
+	DEVICE_MODEL := AP3935
+	SOC := qcom-ipq8068
+	PAGESIZE := 2048
+	BLOCKSIZE := 128k
+	KERNEL = kernel-bin | append-dtb | uImage none | append-uImage-fakehdr filesystem
+	KERNEL_LOADADDR = 0x41408000
+	KERNEL_SIZE := 14400k
+	KERNEL_SUFFIX := -uImage
+	UBINIZE_OPTS := -E 5
+	DEVICE_PACKAGES := ath10k-firmware-qca99x0-ct \
+		ath10k-board-qca99x0-2g-high ath10k-board-qca99x0-5g-high 
```

Comment:
Where are `ath10k-board-qca99x0-2g-high`and  `ath10k-board-qca99x0-5g-high`defined? I can not find them in our codebase.

Reply:
These packages were removed in c2630ac910febd0be4ab8dc41f2aa7bda55e19f4 in favor of `ath10k-board-qca99x0`, this should get fixed.

Reply:
No need for those anymore. Removed.